assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
+
+		&.is-button {
+			background-color: $muriel-white;
+			border: none;
+			border-radius: 3px;
+			cursor: pointer;
+			height: 40px;
```

Comment:
We shouldn't set explicit height or max-height, instead this should be determined by the size of the button + the vertical padding. For the sizing, how about this, all within `&.is-button`:

```
display: block;
height: auto;
max-width: 360px;
width: 100%;
```

Reply:
Yep, these were mostly straight from the Figma dimensions to make sure everything was exactly as presented there. I've tweaked things around a little in [0bd35eb32973dddeab931f11ba4ba1fc61ff7bbc] including padding.